### ISSUE TITLE
Method Html.FrontHtml().PageLink(string,string) should not render attribute 'target' when none is specified.

### ISSUE THREAD

**Issue by MarkSizer** [Kooboo/CMS#241]
When using the method:

   @Html.FrontHtml().PageLink(item.LinkText, item.FullName)

The HTML output fails the W3C validator service with the following message:

   Bad value for attribute target on element a: Browsing context name must be at least one character long.
   <li><a href="/News" target="">News</a></li>
   Syntax of browsing context name or keyword:
   A browsing context name or keyword is either any string that does not start with an underscore (_) or a string that case-insensitively matches one of: _blank, _self, _parent, or _top.

Basically it should not include 'target=""' if a target is not specified.
